docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -883,7 +883,7 @@
             nn.nnn.nnn.111. These are separated with a semi-colon.</para>
 
```

Comment:
I do not see .111 in the image. Look at all the text and make sure it matches what you depict.